init-container/index.js

```diff
@@ -93,10 +93,27 @@ async function innerRun() {
         }
     }
 
+    if (program.decryptedPath == null || program.decryptedPath == "")
+    {
+      throw "decrypted path name wasn't provided altough it's mandataroy";
+    }
+
     checkDirectorySync(program.decryptedPath);
-    
-    const outputFile = path.join(program.decryptedPath, program.decryptedFileName);
-    console.log(`Writing output format using ${program.outputFormat} format to file ${outputFile}`);
+
+    const outputFormat = program.outputFormat.toLowerCase();
+    let outputFile = "";
+
+    if (program.decryptedFileName == null || program.decryptedFileName == "") 
```

Comment:
```suggestion
    if (!program.decryptedFileName) 
```

Reply:
I remembered it's not the same as null or empty, but maybe I was wrong...